save-backend/src/test/resources/application.properties

```diff
@@ -12,9 +12,12 @@ spring.sql.init.mode=always
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.properties.hibernate.use_sql_comments=true
 spring.jpa.properties.hibernate.validator.apply_to_ddl=false
 spring.jpa.hibernate.use-new-id-generator-mappings=false
 # logging
 logging.level.org.cqfn.save.backend.service.TestService=DEBUG
 logging.level.org.cqfn.save.backend.service.TestExecutionService=DEBUG
-logging.level.org.cqfn.save.backend.controllers.TestController=DEBUG
\ No newline at end of file
+logging.level.org.cqfn.save.backend.controllers.internal.TestController=DEBUG
+logging.level.org.hibernate.SQL=DEBUG
```

Comment:
What kind of information we will get by these `debug` and `trace`, I mean will it clutter up our output a lot?

Reply:
Yes, a lot, but I it's only for tests and I thought that sometimes currently logged info (SQL requests with `?` isntead of parameter values) is not enough for debugging

Reply:
ok, didn't pay attention, that it is for tests